### PR TITLE
fix CmAccount instancing in BuyBookingToken function

### PR DESCRIPTION
**Pull request by VjeraTurk**
## Why this should be merged  & How this works
- Fixes using the bot address ( `transportOpts.From`) instead of cmAccount address.
- Skips the `checkAndApproveAllowance` before minting (the approving the allowance is done during the buy operation that is triggered by the `cmAccount.buyBookingToken()` function call).
- Uses `CAM` instead of `EURSH` in `mintnbuy.go` example  
- Add example for Test Token  in `mintnbuy.go` example (with comments)
## How this was tested
Buy running `mintnbuy.go` and via postman using Test Token, CAM and EUR.

### CODE REVIEW

File: pkg/booking/booking.go [hunk 1]
```diff
@@ -132,7 +132,7 @@ func (bs *Service) BuyBookingToken(
 	}
 
 	// Call the BuyBookingToken function from the contract
-	receipt, err := bs.cmAccounts.BuyBookingToken(ctx, bs.transactOpts, tokenID)
+	receipt, err := bs.cmAccounts.BuyBookingToken(ctx, bs.transactOpts, bs.cmAccountAddress, tokenID)
```
Comment (evlekht) [inline pkg/booking/booking.go:127]: if BuyBookingToken is always called with bs.cmAccountAddress, than there is no need to pass it as arg

Comment (VjeraTurk) [reply pkg/booking/booking.go:127]: I would like to introduce new naming e.g.

`myCmAccount` 
 `partnersCmAccount` and  
 `myCmAccountAddress` 
 `partnersCmAccount`
 
 to make it easier to follow which methods need a partner's cmAccount instance and which needs bot's own cmAcocount instance. 
 
 I would not cache myCmAccount, just partner's
 
 What do you think?

Comment (evlekht) [reply pkg/booking/booking.go:127]: ah, wait, I think my comment is invalid, got confused
its already in booking service, its done exactly as I wanted, so, nvm

**Comment by VjeraTurk** [comment]
DONE: Test with test token @nkoljanin

File: pkg/cm_accounts/cm_accounts.go [hunk 2]
```diff
@@ -274,6 +275,7 @@ func (s *service) MintBookingToken(
 		expirationTimestamp,
 		price,
 		paymentToken,
+		false,
```
Comment (havan) [inline pkg/cm_accounts/cm_accounts.go]: Should we put a comment here so it's not forgotten what's this about? Hard coding things always makes me nervous. 😅

**Review by evlekht — DISMISSED** [review]

File: pkg/cm_accounts/cm_accounts.go (hunk 2 above)
Comment (VjeraTurk) [reply pkg/cm_accounts/cm_accounts.go]: Added 2 comments
 https://github.com/chain4travel/camino-messenger-bot/pull/63/files#diff-eaf90c995346ed89c73a0cea15889f6931bf8c34c5765bb16bd709038598fd38R271
https://github.com/chain4travel/camino-messenger-bot/pull/63/files#diff-eaf90c995346ed89c73a0cea15889f6931bf8c34c5765bb16bd709038598fd38R279

**Review by havan — APPROVED** [review]

**Pull request merged**